Libraries/LibWeb/HTML/HTMLTitleElement.cpp

```diff
@@ -50,4 +59,40 @@ void HTMLTitleElement::set_text(String const& value)
     string_replace_all(value);
 }
 
+void HTMLTitleElement::consider_propagate_title_update()
+{
+    if (text().is_empty()) {
+        m_empty_titles_in_a_row++;
+    } else {
+        m_empty_titles_in_a_row = 0;
+    }
```

Comment:
Alternatively, if the text is empty, we could check if the timer is running/active and if not, start it and return from this method. That way, we don't need to count the number of consecutive empty title updates since we know the timer will eventually propagate the title.

Reply:
Not exactly, because two empty updates in a row means the website set the title to be empty and we want to propagate that.

Reply:
hm, right, thinking more about it, it shouldnt matter.